CITATION.cff

```diff
@@ -0,0 +1,124 @@
+cff-version: 1.2.0
+message: "If you use this book, please cite it as below."
+title: "SnowEx Hackweek JupyterBook Tutorials"
+version: 2021.07.23
+date-released: "2021-07-23"
+identifiers:
+- description: "collection of archived snapshots of all versions"
+  type: doi
+  value: TODO
```

Comment:
will fill this in after creating a release, planning to use calendar naming scheme (so 2021.07.23, 2021.09.01, etc. for updated versions)